normits_demand/core/segments.py

```diff
@@ -326,7 +333,10 @@ def _mul_div_segmentation(self,
                 "multiplication.\n"
                 "Expected %s segments.\n"
                 "Found %s segments."
-                % (len(return_seg.segment_names), len(set(multiply_dict.keys())))
+                % (
+                    len(return_seg.segment_names),
+                    len(set(multiply_dict.keys())),
+                )
```

Comment:
I have always assumed there must be a good reason to format like this instead of f strings so I tend to not change when I see them, but will change now!